src/app-layout/visual-refresh-toolbar/state/use-app-layout.tsx

```diff
@@ -377,7 +377,7 @@ export const useAppLayout = (
   useEffect(() => {
     // Close navigation drawer on mobile so that the main content is visible
     if (isMobile) {
-      onNavigationToggle(false);
+      onNavigationToggle({ isOpen: false, autoFocus: false });
```

Comment:
Here and in the effect below we hide the navigation programmatically to optimise UX. In that case the toggle should not get focused. The same is done for visual-refresh and toolbar versions of app layout. The autoFocus is set to true for all other cases (when the panel is opened with a user click or programmatically  from the consumers code).